state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -640,6 +640,10 @@ pub mod pallet {
 		BoostPoolCreated {
 			boost_pool: BoostPoolId<T::TargetChain>,
 		},
+		BoostedDepositLost {
+			prewitnessed_deposit_id: PrewitnessedDepositId,
+			amount: TargetChainAmount<T, I>,
+		},
```

Comment:
Does the id provide enough information to be able to derive a list of impacted accounts?

Reply:
It can link to the DepositBoosted event - which can then be used to derive the impacted accounts